com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Utility/HDRP/Water/EvaluateWaterSimulationCausticsNode.cs

```diff
@@ -0,0 +1,49 @@
+using Usage = UnityEditor.ShaderGraph.GraphDelta.GraphType.Usage;
+
+namespace UnityEditor.ShaderGraph.Defs
+{
+    internal class EvaluateWaterSimulationCausticsNode : IStandardNode
+    {
+        public static string Name => "EvaluateWaterSimulationCaustics";
+        public static int Version => 1;
+
+        public static FunctionDescriptor FunctionDescriptor => new(
+            Name,
+@"Caustics = EvaluateSimulationCaustics(RefractedPositionWS, positionWS, DistordedWaterNDC);",
+            new ParameterDescriptor[]
+            {
+                new ParameterDescriptor("positionWS", TYPE.Vec3, Usage.Local, REF.WorldSpace_Position),
+                new ParameterDescriptor("RefractedPositionWS", TYPE.Vec3, Usage.In),
+                new ParameterDescriptor("DistordedWaterNDC", TYPE.Vec2, Usage.In),
```

Comment:
This is spelled wrong.  It should be DistortedWaterNDC.  Distort is spelled with a T.  This was also wrong in SG1, so it's not your fault. Remember to fix it in the ParameterUIDescriptor and body code also.